Core/ClientSMLSWIG/Python/Python_sml_ClientInterface.i

```diff
@@ -11,6 +11,10 @@
 // handle windows calling convention, __declspec(dllimport), correctly
 %include <windows.i>
 
+%begin %{
+#define Py_LIMITED_API 0x03050000
```

Comment:
Could you add a maintainer note that if this version is changed then the `env["LIMITED_API_TARGET"]` line in SConstruct also has to be updated?